app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java

```diff
@@ -90,4 +94,52 @@ public void onSignUpButtonClicked() {
     public void onTermsAndConditionsClicked() {
         getView().goTermsAndConditions();
     }
+
+    private void getUsersFromServer() {
```

Comment:
This is now better. But check that by doing it like this, you could never do another thing inside the `onResponse(users, response)`. It would be better if you add all inside a `new Callback<List<Users>>` and send it to the enqueue instead of sending `this`.

It would be like:
```
mRetrofitServices.getService(GetCallsAPI.class).getAllUsers().enqueue(new Callback<List<Users>>() {
    @Override
    public void onResponse(Call<List<Users>> call, Response<List<Users>> response) {
        ...
    }

    @Override
    public void onFailure(Call<List<Users>> call, Throwable t) {
        ...
    }
});